### PR TITLE
Feature#122 add scroll to login

### PR DESCRIPTION
**Pull request by Ewar22**
# Description

-add scroll log in

## Screenshots

image
image

### CODE REVIEW

**Review by MaxRetana — APPROVED** [review]
LGTM

**Review by AangelF — APPROVED** [review]

File: app/assets/stylesheets/pages/_users.scss [hunk 1]
```diff
@@ -120,12 +122,23 @@
...
+    padding-top: 1rem;
     flex-wrap: wrap;
   }
+}
+
+@include smallxs {
+  a:last-child{
+    font-size: .7rem;
+  }
 }
```
Comment (juperameza) [inline app/assets/stylesheets/pages/_users.scss:144]: New line at the end of this file

**Review by juperameza — CHANGES_REQUESTED** [review]
Also in my opinion there shouldn't be a scroll on the login, but that's your own design choice

**Review by juperameza — APPROVED** [review]
Lgtm

**Pull request merged**